config/spryk/spryks/Spryker/Glue/AddGlueDeleteResource.yml

```diff
@@ -0,0 +1,60 @@
+spryk: structure
```

Comment:
When testing the following:
```#7 {main}Ahmeds-MacBook-Pro-2:project ahmedsabaa (master) $ vendor/bin/console spryk:run AddGlueDeleteResource
Store: DE | Environment: development
 Enter value for AddGlueDeleteResource.mode argument [project]:
 > core
 Enter value for AddGlueDeleteResource.module argument:
 > AgentsRestApi
 Enter value for AddGlueDeleteResource.organization argument [Spryker]:
 > 
 Enter value for AddGlueDeleteResource.resourceType argument:
 > agents
 Enter value for AddGlueDeleteResource.modelName argument:
 > Agent
 Enter value for AddGlueController.controller argument [AgentsResource]:
 > 
 Enter value for AddGlueProcessorModel.interfaceName argument [{{ className }}Interface]:
 > 
```
I get a model called "AgentResource" seems like a bug to me, also the action added to Controller uses a factory function that does not exist `createAgentDeleter`...seems that interfaces get mixed up too because the model uses an interface with a `configure` function attached to it and not the deleteAgent function or whatever

Reply:
I would also "expect" that by default the model gets the client bridge as a dependency automatically

Reply:
same steps gave me:
``` Enter value for AddGlueDependencyClientBridge.dependentModuleOrganization argument [Spryker]:
 >```

I would assume this would be called `ClientModuleOrganization`? or maybe `clientModule` should have a hint like `Organization.ModuleName`

Reply:
First two comments already resolved, last comment is open